homeassistant/components/alexa/entities.py

```diff
@@ -775,8 +778,9 @@ def default_display_categories(self):
             return [DisplayCategory.MOTION_SENSOR]
         if sensor_type is self.TYPE_PRESENCE:
             return [DisplayCategory.CAMERA]
+        return None
```

Comment:
`default_display_categories` was missing a return statement.
What is meant to happen if it's neither CONTACT, nor MOTION, nor PRESENCE?
Should it return an empty list? or `None`? or is it unreacheable code?